### ISSUE TITLE
"Custom ffmpeg options" window not receiving accessibility focus

### ISSUE THREAD

**Issue by DilsonsPickles** [audacity/audacity#11380]
### Description

When I use keyboard accessibility navigation to open the "Custom ffmpeg options" window, the window does not receive the focus state.

**Comment by Eism** [comment]
@DilsonsPickles where is "Custom ffmpeg options"?

**Comment by DilsonsPickles** [comment]
<img width="636" height="671" alt="Image" src="https://github.com/user-attachments/assets/6d753be7-deef-4383-90d6-2940bff878e4" />

Export modal select "Custom ffmpeg Export" as format and then "Open custom FFmpeg format options" button will show underneath sample rate dropdown.

Open that using keyboard accessibility tabbing and observe.

**Comment by dozzzzer** [comment]
Tested

https://github.com/user-attachments/assets/b2c6aaed-92be-452c-8a06-75903c64d685
